### ISSUE TITLE
[ ERROR ] Calling method '_start_suite' of listener 'DataDriver' failed:

### ISSUE THREAD

**Issue by kstine** [Snooz82/robotframework-datadriver#81]
When running datadriver test case in TRACE logging- seeing this error:
[ ERROR ] Calling method '_start_suite' of listener 'DataDriver' failed: AttributeError: 'keywords' attribute is read-only and deprecated since Robot Framework 4.0. Use 'body', 'setup' or 'teardown' instead.

Does not seem to affect the test execution.

Yes I am running datadriver with pabot:
Library setting:
```robot
Library             DataDriver    file=${FILE_TEST_PATH}    reader_class=csv_reader    arg_name=\${user}
...                     dialect=excel    encoding=utf_8    optimize_pabot=Atomic    config_keyword=Suite Setup Keywords
```
commandline:
```
pabot --pabotlib --processes 10 --testlevelsplit --verbose -d Results -i add-book-perf -L TRACE Tests
```

**Comment by kstine** [comment]
A few updates:
This is using python 3.10 on Windows 10 using venv with VSCode. Does not do anything particularly harmful, and only noticeable when logging set to TRACE.

**Comment by Snooz82** [comment]
That "error" only happened with Pabot. It had not a real impact, because it just removed all content from the initial Suite started by pabot, so that it is skipped.

However: TD;DR Fixed in 1.7.0
